.github/workflows/release.yml

```diff
@@ -41,43 +19,47 @@ jobs:
 
     runs-on: ${{ matrix.os }}
     steps:
-      # stack was removed in macOS-14 which is now latest
-      # https://discourse.haskell.org/t/github-hosted-runner-for-macos-aarch64/8717/16
-      - if: ${{ runner.os == 'macOS' }}
-        run: curl -sSL https://get.haskellstack.org/ | sh
-
-      - uses: actions/checkout@v4
-      - uses: freckle/stack-cache-action@v2
-      - run: echo "$HOME/.local/share/gem/ruby/3.0.0/bin" >>"$GITHUB_PATH"
-      - run: gem install --user ronn-ng
       - if: ${{ runner.os == 'macOS' }}
         run: brew install coreutils # need GNU install
+      - run: gem install --user ronn-ng
+      - run: |
+          for bin in "$HOME"/.local/share/gem/ruby/*/bin; do
+            echo "$bin"
+          done >>"$GITHUB_PATH"
+      - uses: actions/checkout@v4
+      - uses: freckle/stack-action@v5
```

Comment:
is the stack-action here replacing the 'install stack' stuff that was removed?

Reply:
`stack-action` does have an "install stack if needed", so moving to it does achieve that (among other things).